packages/javascript-sdk/src/fr-webauthn/fr-webauthn.mock.data.ts

```diff
@@ -8,11 +8,13 @@
  * of the MIT license. See the LICENSE file for details.
  */
 
+import { CallbackType } from '../auth/enums';
+
 export const webAuthnRegJSCallback653 = {
   authId: 'eyJ0eXAiOiJKV1QiLCJhbGciOiJIUzI1NiJ9 ... ',
   callbacks: [
     {
-      type: 'TextOutputCallback',
+      type: CallbackType.TextOutputCallback,
```

Comment:
Not sure why this wasn't caught before, but we needed the proper types on this mock data.

Reply:
This must have snuck in from me doing side work and somehow never got caught, unsure how it ended up here but good catch